python/turbine_models/tests/sd_test.py

```diff
@@ -81,42 +86,49 @@ def testExportClipModel(self):
         )
         err = utils.largest_error(torch_output, turbine[0])
         assert err < 9e-5
-        os.remove("stable_diffusion_v1_4_clip.safetensors")
-        os.remove("stable_diffusion_v1_4_clip.vmfb")
+        # os.remove(f"{arguments['safe_model_name']}_clip.safetensors")
+        # os.remove(f"{arguments['safe_model_name']}_clip.vmfb")
```

Comment:
Why are these commented out?

Reply:
Some race condition between this and the actual test execution. Will leave these comments unresolved until test file mgmt is fixed.